content/en/events/kcseu/faq.md

```diff
@@ -74,7 +79,7 @@ contact us at community@kubernetes.io about registering.
 If you are a SIG lead or subproject owner and there is someone that is not yet
 an org member or from another project that might take part in a panel,
 SIG face-to-face meeting etc, you can sponsor them to attend the summit.
-Please reach out to community@kubernetes.io to make arrangements.
+Please [email us] to make arrangements.
```

Comment:
```suggestion
Please [email us](community@kubernetes.io) to make arrangements.
```

Or is there some special kind of markdown replacement in place to set the email?

Reply:
Yes, there is, see the bottom of the doc.